### PR TITLE
Package and release the Servo magicleap app

### PR DESCRIPTION
**Pull request by asajeffrey**
Package the Magic Leap Servo2D app, and build and release it nightly.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes do not require tests because we don't test ML

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22180)

### CODE REVIEW

File: etc/ci/buildbot_steps.yml [hunk 1]
```diff
@@ -232,11 +232,31 @@ android-nightly:
...
   MAGICLEAP_SDK: /Users/servo/magicleap/v0.17.0
+  MLCERT: /Users/servo/magicleap/certs/TempSharedCert.cert
  commands:
+  - unset OPENSSL_INCLUDE_DIR
+  - unset OPENSSL_LIB_DIR
   - ./mach clean-nightlies --keep 3 --force
   - ./mach clean-cargo-cache --keep 3 --force
   - ./etc/ci/clean_build_artifacts.sh
   - ./mach build --magicleap --dev
+  - /Users/servo/magicleap/v0.17.0/mabu -o ./target/release/magicleap -t lumin_debug ./support/magicleap/Servo2D/Servo2D.package
```
Comment (jdm) [inline etc/ci/buildbot_steps.yml]: I would rather have a `./mach package --magicleap` command here.

**Review by jdm — CHANGES_REQUESTED** [review]

File: etc/ci/buildbot_steps.yml (hunk 1 above)
Comment (asajeffrey) [reply etc/ci/buildbot_steps.yml]: I was going to hold off on a proper mach package command until we've worked on the app UI a bit more. Not a huge deal though.

Comment (jdm) [reply etc/ci/buildbot_steps.yml]: I want it to be as easy as possible to create a package to distribute to partners or for local testing.

Comment (asajeffrey) [reply etc/ci/buildbot_steps.yml]: OK, it should be pretty easy to add to the package commands.

Comment (asajeffrey) [reply etc/ci/buildbot_steps.yml]: Serves me right for saying "easy".

**Review by jdm — APPROVED** [review]

**Comment by asajeffrey** [comment]
Squashed. @bors-servo r=jdm

**Comment by asajeffrey** [comment]
Lets see if our CI is working again... @bors-servo retry

**Pull request merged**